index.js

```
@@ -1,51 +1,98 @@
+
+// Keep track of which canvases have been transferred
+const transferredCanvases = new WeakSet()
+
+// Clean up worker for a canvas
+function cleanupWorker(canvas) {
+  if (canvas.worker) {
+    // Send cleanup message to worker before terminating
+    canvas.worker.postMessage({ type: "cleanup" })
+    // Remove from transferred set
+    transferredCanvases.delete(canvas)
+    delete canvas.worker
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)